tests/application/assets/classes/TestDriver.php

```diff
@@ -13,10 +13,16 @@ public function getAdminAreas()
     {
         return array(
             'devel' => 'Devel',
-            'settings' => 'Settings'
+            'settings' => 'Settings',
+            'wizardtest' => 'WizardTest'
         );
     }
 
+    public function redirectTo($paramsOrURL = null)
```

Comment:
Add the `:void` type hint here, and add a PHPDoc for the method: This is important because the base method has `@return never`, which is a big change.